docs/register-api-plugins.md

```diff
@@ -0,0 +1,169 @@
+### Register API Plugins
+
+The Reaction API server has a plugin system that allows code to be broken into small packages. Plugins can register functions, configuration, and GraphQL schemas and resolvers.
+
+Some API plugins are included in the API codebase, but in general you should create plugins as NPM packages. This does not necessarily mean they need to be published to NPM, but they must be something you can add to `package.json` and NPM will know how to install it.
+
+At a high level, an API plugin package is one where the main export is an async function that accepts a `ReactionAPI` instance and calls `api.registerPlugin` before returning. In a Reaction API project, your plugin package will be imported and used like this:
+
+```js
+import registerSomePlugin from "some-plugin";
```

Comment:
Why is this `some-plugin`? That seems confusing

Reply:
update to "your-plugin"